executor-scripts/linux/forward

```diff
@@ -0,0 +1,19 @@
+#!/bin/sh
+
+yesno() {
+	case "$1" in
+	yes|1)	echo 1 ;;
+	*)	echo 0 ;;
+	esac
+}
+
+[ "$PHASE" != "up" ] && exit 0
+[ -z "$VERBOSE" ] || set -x
+
+[ -n "$IF_FORWARD_IPV4" ] && ${MOCK} /bin/sh -c "echo $(yesno $IF_FORWARD_IPV4) > /proc/sys/net/ipv4/conf/$IFACE/forwarding"
```

Comment:
How 'bout `sysctl net.ipv4.conf.$IFACE.forwarding=1`?

Reply:
In general, I would prefer the executors to work without sysctl(8) being present.  That is how the other executors are written.  If we want to change that assumption, we should do so in another issue to discuss the pros and cons.